src/plans/plan-interface.h

```diff
@@ -89,20 +44,29 @@ LibraryAction DeliverSample (In Real X,
                              In Real Y,
                              In Real Z);
 
-// Does nothing
-LibraryAction Stub (In String desc);
-
 
-// Lookups
-
-// Can be used for fine-grain process control, but generally not needed.
-Boolean Lookup Running (String operation_name);
-Boolean Lookup Finished (String operation_name);
+// Lander queries
 
 Boolean Lookup HardTorqueLimitReached (String joint_name);
 Boolean Lookup SoftTorqueLimitReached (String joint_name);
+Boolean Lookup ImageReceived;
```

Comment:
Is this still a valid flag. I don't see where it is being set or consumed by any of the current example plans.

Reply:
Forgot to remove - gone now.